standalone-metastore/metastore-server/src/test/java/org/apache/hadoop/hive/metastore/utils/TestTxnDbUtil.java

```diff
@@ -370,6 +384,8 @@ public static String queryToString(Configuration conf, String query, boolean inc
     return sb.toString();
   }
 
+
```

Comment:
nit: is this needed? we already have 1 line space between the methods

Reply:
Thank you for spotting. It was a mistake. I kept one line as a spacer between this and the next method.